packages/server-side-render/src/server-side-render.js

```diff
@@ -39,11 +39,20 @@ function DefaultErrorResponsePlaceholder( { response, className } ) {
 	return <Placeholder className={ className }>{ errorMessage }</Placeholder>;
 }
 
-function DefaultLoadingResponsePlaceholder( { className } ) {
+function DefaultLoadingResponsePlaceholder( { children } ) {
```

Comment:
I think removing the `className` prop will make this a breaking change. We should avoid that.